tox.ini

```diff
@@ -16,6 +16,16 @@ commands =
 [testenv:ansible-lint]
 commands =
     {toxinidir}/scripts/linting-ansible.sh
+deps=
+    -rtest-requirements.txt
+    ansible>=1.9.1,<2.0.0
+
+[testenv:ansible2-lint]
+commands =
+    {toxinidir}/scripts/linting-ansible.sh
```

Comment:
This can be (more simply)

```
commands = {[testenv:ansible2-lint]commands}
```

That way if one of them changes, they both change automatically.